### DISCUSSION TITLE
DNS_PROBE_STARTED

### DISCUSSION THREAD

**Discussion by kevinsdi** [plausible/analytics#1756] [Anything Else]
Why the website is not opening for me? 
Browser says:
plausible.io’s DNS address could not be found. Diagnosing the problem.
DNS_PROBE_STARTED

**Comment by metmarkosaric** [comment]
thanks for reporting that @kevinsdi! sounds weird! everything seems to be working fine here and we have no other reports of this. could it be that you're running some type of adblocker that's blocking our website?

**Reply by kevinsdi** [reply]
It is not working from Chrome, Safari, Brave Browser, Firefox, and it is not depends on any browser extension, but its working in Tor Browser

**Reply by kevinsdi** [reply]
And it worked 5-7 days ago with no problems

**Reply by Vigasaurus** [reply]
Sounds like your DNS resolver is the problem then (it applies for the whole OS, thus all browsers would be affected, except Tor which forcibly changes it). I'd say you should try changing the DNS server your computer is using from the current to something like Cloudflare's 1.1.1.1 or Quad9's 9.9.9.9 and see if it works then (it should, in theory). You'd need to find instructions to change the DNS server based on your OS/browser of choice though (I believe all the major browsers to offer DNS over HTTPS now, and that's usually easier than changing the OS-level setting).

**Reply by kevinsdi** [reply]
The problem is in my VPN)) It's working fine without VPN 🤷‍♂️
